context/pom.xml

```diff
@@ -39,5 +39,11 @@
       <artifactId>junit-jupiter-api</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <version>5.18.0</version>
```

Comment:
Version should go to top level pom depMgt section. No version should be in lower poms.